src/store/modules/position.store.js

```diff
@@ -297,6 +319,14 @@ const actions = {
     setCameraPosition({ commit }, cameraPosition) {
         commit('setCameraPosition', cameraPosition)
     },
+    setProjection({ commit }, projection) {
+        if (
+            projection instanceof CoordinateSystem &&
+            (projection.epsg === LV95.epsg || projection.epsg === WEBMERCATOR.epsg)
+        ) {
+            commit('setProjection', projection)
+        }
+    },
```

Comment:
What about other projections ?

Reply:
I've added some logging in case an unwanted projection happens to come here